### PR TITLE
feat(rdc): Add Meridian RDC flow to dvsim

### PR DESCRIPTION
**Pull request by eunchan**
This commit adds meridian rdc skelecton flow to dvsim.

It scaffolds the environment from CDC environment as they are similar.
RdcCfg has been added to dvsim but parsing the result is not ready yet.

### CODE REVIEW

File: hw/rdc/tools/meridianrdc/run-rdc.tcl [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+  }
+}
+
+# Report waived in a separate file
+report_policy -verbose -skip_empty_summary_status -compat -output ../REPORT/mrdc.rpt {NEW TO_BE_FIXED DEFERRED}
+report_policy -verbose -skip_empty_summary_status -compat -output ../REPORT/mrdc.waived.rpt {WAIVED}
+
+# report_messages -output mrdc.rpt
+
+# Clock report
```
Comment (msfschaffner) [inline hw/rdc/tools/meridianrdc/run-rdc.tcl:187]: Is this part here still needed?

File: hw/top_earlgrey/rdc/chip_earlgrey_asic_rdc_cfg.hjson [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+  ]
+
+  // Timing constraints for this module
+  sdc_file: "{proj_root}/hw/top_earlgrey/syn/chip_earlgrey_asic.sdc"
+
+  // Verix environment file with additional definitions (may have to populate later)
+  env_file: ""
+
+  // Main CDC waiver file
```
Comment (msfschaffner) [inline hw/top_earlgrey/rdc/chip_earlgrey_asic_rdc_cfg.hjson:30]: out of date comment?

**Review by msfschaffner — APPROVED** [review]
This LGTM, thanks for adding it!

You may also want to update this part here in Dvsim to reflect that `mrdc` is now supported as well:
https://github.com/lowRISC/opentitan/blob/db28d6d3b29bd99de8680dbd476ebdc88927186b/util/dvsim/dvsim.py#L297-L298

**Review by jeoongp — APPROVED** [review]
LGTM!

**Comment by eunchan** [comment]
This shouldn't affect CW310 tests. I will merge this.
image

**Pull request merged**